queue_services/entity-emailer/tracker/config.py

```diff
@@ -58,7 +58,9 @@ class _Config():  # pylint: disable=too-few-public-methods
     """
 
     # PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
-
+    
+    # used to identify versioning flag
+    SERVICE_NAME = 'emailer'
```

Comment:
As per discussion using flag `emailer` for the tracker. https://github.com/bcgov/lear/pull/3092#discussion_r1859101021